cea/resources/radiation_arcgis/calculate_sunny_hours_of_year.py

```diff
@@ -54,16 +54,11 @@ def calculate_sunny_hours_of_day(day, sunrise, temporary_folder):
     # Counter of Columns in the Initial Table
     Counter = radiation_sunnyhours.count(1)[0]
     values = Counter - 1
-    # Condition to take into account daysavingtime in Switzerland as the radiation data in ArcGIS is calculated for 2013.
-    if 90 <= day < 300:
-        D = 1
-    else:
-        D = 0
     # Calculation of Sunrise time
     Sunrise_time = sunrise[day - 1]
     # Calculation of table
     for x in range(values):
-        Hour = int(Sunrise_time) + int(D) + int(x)
+        Hour = int(Sunrise_time) + int(x)
```

Comment:
While not really a problem, it is slightly faster to not do the casting (`int(Sunrise_time)`):
```
In [4]: s = 3

In [5]: %%timeit
   ...: for x in range(1000):
   ...:    hour = s + x
   ...:
10000 loops, best of 3: 43.7 µs per loop

In [6]: %%timeit
   ...: for x in range(1000):
   ...:    hour = int(s) + int(x)
   ...:
1000 loops, best of 3: 246 µs per loop
```

Reply:
it is needed, so better we keep it.